makefile

```diff
@@ -15,21 +15,28 @@ GDC ?= gdc
 
 dmd: bin/dfmt
 
-ldc: $(SRC)
+ldc: $(SRCS)
 	$(LDC) $(LDC_FLAGS) $^ -ofbin/dfmt
 	-rm -f *.o
 
-gdc: $(SRC)
+gdc: $(SRCS)
 	$(GDC) $(GDC_FLAGS) $^ -obin/dfmt
 
 test: debug
 	cd tests && ./test.sh
 
-bin/dfmt-test: $(SRC)
+bin/dfmt-test: $(SRCS)
 	$(DC) $(DMD_TEST_FLAGS) $^ -of$@
 
-bin/dfmt: $(SRC)
+bin/dfmt: $(SRCS)
 	$(DC) $(DMD_FLAGS) $^ -of$@
 
-debug: $(SRC)
+debug: $(SRCS)
```

Comment:
You don't really need to change all these `$(SRC)` to `$(SRCS)` now that makd is not directly included.